core/src/main/java/org/wildfly/channel/proxy/HttpProxy.java

```diff
@@ -0,0 +1,54 @@
+package org.wildfly.channel.proxy;
+
+import org.apache.http.HttpHost;
+import org.jboss.logging.Logger;
+
+import java.net.URL;
+
+public abstract class HttpProxy {
+
+    private static final Logger LOG = Logger.getLogger(HttpProxy.class);
+
+    private HttpProxy() {
+    }
+
+    public static String PROXY_AUTHORIZATION = "Proxy-Authorization";
```

Comment:
Could you please add javadoc all these constants?

Reply:
I also add final, so I think they start to appear in the "Constant Field Values" table of the javadocs